keanu-project/src/main/java/io/improbable/keanu/vertices/VertexId.java

```diff
@@ -84,4 +84,8 @@ public int getIndentation() {
     public long[] getValue() {
         return Arrays.copyOf(idValues, idValues.length);
     }
+
+    public static void resetIdGenerator() {
+        ID_GENERATOR.set(0);
```

Comment:
Were we setting this to `1` for any particular reason before?

Reply:
Not really - it was just a good known value - I've decided to set it 0 everywhere just as that's the value we start at normally so it seems more consistent.